planet/models.py

```diff
@@ -157,16 +156,6 @@ def num_bytes_downloaded(self):
         '''
         return self.response.num_bytes_downloaded
 
-    def last_modified(self):
```

Comment:
I believe I was wrong. I think we should keep this function. Although this planet module doesn't call for this function, `last_modified()`, a user might actually want to use this function. So, I suggest putting it back in along with the `datetime` module.

Now, my first comment still stands - we need to either change `datetime.strptime()` to `datetime.datetime.strptime()`. Or you can do `from datetime import datetime`, then `datetime.strptime()` will work :)